functions/AutoRemediateCFM-005.js

```diff
@@ -1,13 +1,12 @@
-'use strict'
 
-const AWS = require('aws-sdk')
+const { CloudFormationClient, UpdateTerminationProtectionCommand } = require('@aws-sdk/client-cloudformation')
 
 /**
  * Lambda function to enable CloudFormation Stack Termination Protection
  *
  */
 
-module.exports.handler = (event, context, callback) => {
+const handler = async (event, context, callback) => {
```

Comment:
⛏️  This applies to all the files. A find and replace should make this a pretty simple change
```suggestion
const handler = async (event) => {
```

Reply:
it is a much bigger change than you think, i also will need to replace all of the following too
`callback(err, 'failed to auto-remediate', CCRuleCode)`
`callback(null, 'Success')`
and update all tests again

Reply:
Now that all the handlers have been updated to `return` and `throw` this comment can be actioned